Client/Frontend/Reader/ReaderModeCache.swift

```diff
@@ -35,8 +37,9 @@ class ReaderModeCache {
 
     func get(url: NSURL) throws -> ReadabilityResult {
         let error: NSError! = NSError(domain: ReaderModeCacheErrorDomain, code: ReaderModeCacheErrorCode.NoPathsFound.rawValue, userInfo: nil)
```

Comment:
That doesn't handle the case where `ReadabilityResult(string: string as String)` returns `nil`, though.